meecrowave-specs-api/pom.xml

```diff
@@ -206,6 +206,126 @@
             </transformers>
           </configuration>
         </execution>
+        <execution>
+          <id>jakarta</id>
+          <phase>package</phase>
+          <goals>
+            <goal>shade</goal>
+          </goals>
+          <configuration>
+            <shadedArtifactAttached>true</shadedArtifactAttached>
+            <shadedClassifierName>jakarta</shadedClassifierName>
+            <createDependencyReducedPom>false</createDependencyReducedPom>
+            <transformers>
+              <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer" />
+              <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+              </transformer>
+              <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
```

Comment:
guess cxf and log4j config is not needed here?
Also relocation patterns are too complete (persistence, validation etc are not provided)

Reply:
Thanks, updated.